js/classes/entities/midground/enemies/flying_enemy_base.js

```diff
@@ -81,4 +94,33 @@ class FlyingEnemyBase {
             this.path[this.target], this.enemy.pos)), this.enemy.speed * GAME.clockTick);
         this.enemy.pos = Vector.add(this.enemy.pos, velocity);
     }
+
+
+
+    // presets for paths
+
+    static get STRAIGHT_LINE() {
+        return [new Vector(0, 0), new Vector(400, 0)];
+    }
+
+    static get ZIG_ZAG() {
+        return [new Vector(0, 0), new Vector(200, 200), new Vector(400, 0), new Vector(200, -200), new Vector(0, 0)];
```

Comment:
What does this formatting do? How is this used?

Reply:
What do the values in this array do?

Reply:
It's a lot like my "presets" I made for particle effects. ZIG_ZAG is just an object containing a bunch of vectors to represent a path, specifically a path that forms a zig zag pattern.